Example/Example/AppDelegate.swift

```diff
@@ -134,30 +134,30 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
             print("Matrix: status = \(Radar.stringForStatus(status)); matrix[0][0].duration.text = \(String(describing: matrix?.routeBetween(originIndex: 0, destinationIndex: 0)?.duration.text)); matrix[0][1].duration.text = \(String(describing: matrix?.routeBetween(originIndex: 0, destinationIndex: 1)?.duration.text)); matrix[1][0].duration.text = \(String(describing: matrix?.routeBetween(originIndex: 1, destinationIndex: 0)?.duration.text)); matrix[1][1].duration.text = \(String(describing: matrix?.routeBetween(originIndex: 1, destinationIndex: 1)?.duration.text))")
         }
 
-        Radar.sendEvent(customType: "custom_event", metadata: ["data": "test"]) { (status, location, events, user) in
+        Radar.logConversion(name: "custom_event", metadata: ["data": "test"]) { (status, location, events, user) in
```

Comment:
just want to confirm cc @tjulien - can we remove the old signature completely? May have to deprecate the `sendEvent` method if there's callers in the wild so we don't break, but if not can remove

Reply:
Yeah, we’re going to remove sendEvent. We’ll put it in the Migration guide